config/routes.rb

```diff
@@ -3,6 +3,8 @@
 
   resources :proposals, except: :delete do
     resources :experiments, except: [:delete, :index] do
+      resources :procedures, only: [:show, :new, :create, :edit, :update]
```

Comment:
use except: [:delete, :index] instead of only, that way you can specify less parameters